tests/test_routes.py

```diff
@@ -0,0 +1,19 @@
+# test that app returns empty list when db is empty
+def test_get_all_planets_with_no_records(client):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_one_planet_with_two_records(client, two_saved_planets):
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == {
+        "id": 1,
+        "name": "Earth",
+        "description": "Its a planet",
+        "distance_from_sun": 92
+    }
```

Comment:
Consider checking for each key/value pair separately to make it easier to read any test failures:

```suggestion
    assert response_body["id"] == 1
    assert response_body["name"] == "Earth"
    assert response_body["description"] == "Its a planet"
    assert response_body["distance_from_sun"] == 92
```